project_one/lib/project_one.ex

```diff
@@ -0,0 +1,33 @@
+defmodule ProjectOne do
+  @moduledoc """
+  Documentation for `ProjectOne`.
+  """
+
+  @doc """
+  Hello chmod.
+
+  ## Examples
+
+      iex> ProjectOne.calculate_chmod(%{user: "rwx", group: "r-x", other: "r-x"})
+      "755"
+
+  """
+  def calculate_chmod(perm) do
```

Comment:
А как же пасхальные яйца?
```suggestion
  def calculate_chmod("bond"), do: "007"
  def calculate_chmod(perm) do
```